.github/workflows/sdk-breaking-change-labels.yaml

```
@@ -0,0 +1,57 @@
+name: SDK Breaking Change Labels (Preview)
+
+on:
+  check_run:
+    types: [completed]
+  workflow_dispatch:
+    inputs:
+      check_run_id:
+        description: 'Check run ID to process'
+        required: false
+        type: string
+
+permissions:
+  contents: read
+
+jobs:
+  sdk-breaking-change-labels:
+    name: SDK Breaking Change Labels (Preview)
+    runs-on: ubuntu-24.04
```

```suggestion
    if: contains(github.event.check_run.name, '(SDK Generation Job)') && github.event.check_run.check_suite.app.name == 'Azure Pipelines`
    runs-on: ubuntu-24.04
```

Filter as aggressively as possible based on the properties of `github.event` before allocating an agent.

---

Yes if we can do that level of filtering we should.

---

Updated.